src/mapgen/mapgen_v7.h

```diff
@@ -1,7 +1,7 @@
 /*
 Minetest
-Copyright (C) 2013-2018 kwolekr, Ryan Kwolek <kwolekr@minetest.net>
```

Comment:
This should stay at 2018, shouldn't it?

Reply:
Good question, i am not sure if copyright dates should cover the times of contributions, or cover up to the present.

Reply:
Time of contributions is also how it's been handled in other files, I think.

Reply:
Yes i think so too.
Searching shows kwolekr's last commits were in 2016 so used that.